src/typinf.c

```diff
@@ -376,7 +379,10 @@ class TypeInfoDtVisitor : public Visitor
         assert(name);
         size_t namelen = strlen(name);
         dtsize_t(pdt, namelen);
-        dtabytes(pdt, 0, namelen + 1, name);
+        dtxoff(pdt, d->csym, Type::typeinfofunction->structsize);
```

Comment:
We could avoid the relocations by turning name into a function. I think we do this for ModuleInfo.